src/electron/package_common_action.sh

```diff
@@ -14,18 +14,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-yarn do src/electron/linux/build
+yarn do src/electron/build
```

Comment:
Can't we move everything here to src/electron/build? Why do we need `package_common_action.sh`?

Reply:
Oh, for faster builds. The icon step, in particular, takes several seconds.